api/tbcapi/tbcapi.go

```diff
@@ -29,9 +32,43 @@ type (
 	PingResponse protocol.PingResponse
 )
 
+type BtcHeader struct {
+	Version uint32 `json:"version"`
+
+	// hex encoded byte array
+	PrevHash string `json:"prev_hash"`
+
+	// hex encoded byte array
+	MerkleRoot string `json:"merkle_root"`
+
+	Timestamp uint32 `json:"timestamp"`
```

Comment:
I think we could use an `uint64` for the timestamp. `uint32` is okay, but I am generally against using 32-bit integers for storing time.

Reply:
I can update this, it's pretty quick 👍